templates/strings/objc-m.stencil

```diff
@@ -0,0 +1,64 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+
+#import "Localizable.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-security"
+
+NSString* tr(NSString* key, ...) {
+    va_list args;
+    va_start(args, key);
+
+    NSString *format = [NSBundle.mainBundle localizedStringForKey:key value:@"" table:nil];
+    NSString *result = [[NSString alloc] initWithFormat:format arguments:args];
+    return result;
+};
+
```

Comment:
You probably want a `#pragma clang diagnostic pop` around here?

Reply:
agreed